addons/mrp/models/mrp_unbuild.py

```diff
@@ -119,12 +119,16 @@ def action_unbuild(self):
         for produce_move in produce_moves:
             if produce_move.has_tracking != 'none':
                 original = original_quants.filtered(lambda quant: quant.product_id == produce_move.product_id)
-                self.env['stock.move.lots'].create({
-                    'move_id': produce_move.id,
-                    'lot_id': original.lot_id.id,
-                    'quantity_done': produce_move.product_uom_qty,
-                    'quantity': produce_move.product_uom_qty
-                })
+                group_by_lot = {}
+                for o in original:
+                    group_by_lot.setdefault(o.lot_id.id, 0) += produce_move.product_uom_qty
```

Comment:
as the move lots are linked to the same move, the quantity on the move is the sum of the move lots.  So, if you put multiple move lots, that would go against that.